src/index.html

```diff
@@ -11,6 +11,48 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="searchForm"
+      data-qa="big"
+    >
+      <label>
+        <img
+        src="./images/Search.svg"
+        alt="search"
+        class="icon"
+      >
+        <input
+          type="text"
+          name="search"
+          class="input bigInput"
```

Comment:
```suggestion
          class="input"
```
checklist p.2 https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md